metaseq/modules/activation_functions.py

```diff
@@ -30,6 +30,17 @@ def gelu_back(g, x):
     return ff * g
 
 
+@torch.jit.script
+def relu(x):
+    return F.relu(x)
+
+
+@torch.jit.script
+def relu_back(g, x):
+    g[x <= 0] = 0
+    return g
```

Comment:
is it okay to be modifying g inline?

Reply:
I guess it is because we only use grad_fc2_input for this computation and that's it. Still a bit scary